src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+	import Button from '$lib/components/ui/Button.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { DApp } from '$lib/types/dapp';
+	export let dApp: DApp;
+</script>
+
+<div
+	class="flex h-64 items-end overflow-hidden rounded-2xl bg-cover bg-center"
+	style="background-image: url('{dApp.imageUrl}')"
+>
+	<div class="flex-1 px-4 py-4 backdrop-blur-sm">
+		<div class="flex items-center gap-x-2">
+			<img class="h-12 w-12 rounded-full" src={dApp.logoUrl} alt={dApp.name} />
+			<div class="flex-1">
+				<h6 class="text-sm font-bold text-info">{$i18n.dapps.text.featured}</h6>
```

Comment:
Mostly because I think it might make sense looking at the overall page. But I have no hard feelings changing it to p / div. Whatever you prefer. 

<img width="429" alt="image" src="https://github.com/user-attachments/assets/81924fa1-7d5a-4fee-b8de-1ffc2ab67840">

Reply:
There are better specialists in nomenclature at the foundation, like @meodai, but maybe for the banner an `h2` for `KongSwap` and some sort of tag for its `Featured`, assuming that after the title it's the most important information? But yeah not sure. Not a blocker.